### PR TITLE
Dashboard: Auto plot last result

### PR DESCRIPTION
**Pull request by urig**
If you choose an experiment that does not have any plots associated with saved in the DB, the dashboard will now try to fetch the last result saved for that experiment and plot it automatically.

### CODE REVIEW

File: entropylab/results/dashboard/auto_plot.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+import numpy as np
+import plotly.graph_objects as go
+import plotly.express as px
+
+from entropylab.api.errors import EntropyError
+from entropylab.api.plot import CirclePlotGenerator
+
+
+def auto_plot(data) -> go.Figure:
```
Comment (qguyk) [inline entropylab/results/dashboard/auto_plot.py:11]: please make this function not implementation aware, so it returns the data and generator and plotting done outside

File: entropylab/results/dashboard/dashboard_data.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import abc
 from typing import List
 
-import numpy as np
+# import numpy as np
```
Comment (qguyk) [inline entropylab/results/dashboard/dashboard_data.py:4]: it's better to remove the old code and imports

**Review by qguyk — CHANGES_REQUESTED** [review]

**Comment by qguyk** [comment]
@urig please fix the build checks

File: entropylab/results/dashboard/auto_plot.py (hunk 1 above)
Comment (urig) [reply entropylab/results/dashboard/auto_plot.py:11]: I'll move it to the `DashboardDataReader` as originally planned.

Comment (urig) [reply entropylab/results/dashboard/auto_plot.py]: Done. Kindly review in `dashboard_data.py`.

**Review by qguyk — DISMISSED** [review]

**Review by qguyk — APPROVED** [review]

**Pull request merged**